packages/components/atoms/f-form-field/src/components/_tests/FormDropdown.test.js

```diff
@@ -67,7 +67,7 @@ describe('FormDropdown', () => {
         });
 
         describe('value ::', () => {
-            it('should default to first dropdown option when omitted', () => {
+            it.skip('should default to first dropdown option when omitted', () => {
```

Comment:
What was the reason for this test being skipped?

Reply:
ah yes these were the last remaining tests that I'd skipped 😂 error for this one is below - I'm unsure why this is failing since the component does default to the first option, so it shouldn't be blank - but I'll have another look now. If not, might be worth keeping skipped and I can revisit in a quick follow-up PR
<img width="1084" alt="Screenshot 2024-03-13 at 11 01 46" src="https://github.com/justeattakeaway/fozzie-components/assets/49618712/0addaeba-91ed-4d87-a31c-f1739dcee4c2">

Reply:
fixing forward with this and the below test in a follow up PR 👍